modules/viz/include/opencv2/viz/types.hpp

```diff
@@ -119,6 +119,12 @@ namespace cv
         class CV_EXPORTS Mesh
         {
         public:
+            enum FileTypeReader
+            {
+                PLY,
+                OBJ
+            };
```

Comment:
Do you have a compilation issue? You may throw non-supported exception, this is not a critical feature.